pydoctor/epydoc/markup/__init__.py

```diff
@@ -276,13 +276,14 @@ def __repr__(self) -> str:
 ##################################################
 ## Docstring Linker (resolves crossreferences)
 ##################################################
-class DocstringLinker:
+class DocstringLinker(abc.ABC):
```

Comment:
This seems like a step in the right direction (at least it doesn't pretend to be concrete any more!) but could this be a `Protocol` some day?